documentation/plain_rules.md

```diff
@@ -173,31 +182,68 @@ It could happen (in theory) that a just got auth token also produce a 401 Not au
 
 
 ### HTTP POST action
-Makes an HTTP POST to an URL specified in `url` inside `parameters`, sending a body built from `template`.
+Makes an HTTP POST to an URL specified in `url` inside `parameters`, sending a body built from `template`. 
+The `parameters` field can specify
+* method: *optional*, HTTP method to use, POST by default
```

Comment:
It is funny to see a section named "HTTP POST action" that at the end could not be a POST :)

Maybe the section name (along with all the references to it in other places in the documentation) should be generalized to "HTTP request action".